service/templates/cloudformation/host/main.yaml

```diff
@@ -0,0 +1,24 @@
+{{define "main"}}AWSTemplateFormatVersion: 2010-09-09
+Description: Main CloudFormation stack.
+Resources:
+  PeerRole:
+    Type: 'AWS::IAM::Role'
+    Properties:
+      RoleName: Peer-{{ .ClusterID }}
```

Comment:
Should this be using the key function?

Reply:
Yep, I'll add it to the rest of the host stack changes.